### PR TITLE
fixed to create a combined file, even if the classic output is missing… 

### PR DESCRIPTION
**Pull request by golnazads**
…or empty, adjusted the unittests. Also renamed status_code to status_flaw, since it was more than a code info in that field

### CODE REVIEW

**Comment by seasidesparrow** [comment]
Quick favor: would you please update requirements.txt to use ADSGoogleConnector.git@v0.0.3 instead of v0.0.1?

**Comment by golnazads** [comment]
Done.

File: adsdocmatch/match_w_metadata.py [hunk 1]
```diff
@@ -158,10 +158,10 @@ def match_to_arXiv(self, filename):
                 elif status == 0:
                     return [{'source_bibcode': metadata.get('bibcode'), 'comment': 'from JournalDB: do not match.'}]
                 elif status == -1:
-                    return [{'source_bibcode': metadata.get('bibcode'), 'comment': 'from JournalDB: did not recognize the bibcode.', 'status_code': 'unrecognizable bibstem, processing stopped, shall be added to the rerun list.'}]
+                    return [{'source_bibcode': metadata.get('bibcode'), 'comment': 'from JournalDB: did not recognize the bibcode.', 'status_flaw': 'unrecognizable bibstem, processing stopped, shall be added to the rerun list.'}]
         except Exception as e:
             logger.error('Exception: %s'%e)
-            return [{'source_bibcode': metadata.get('bibcode'), 'comment' : 'Exception: %s in metadata file: %s'%(e, filename), 'status_code' : 'got exception, processing stopped, shall be added to the rerun list.'}]
+            return [{'source_bibcode': metadata.get('bibcode'), 'comment' : 'Exception: %s in metadata file: %s'%(e, filename), 'status_flaw' : 'got exception, processing stopped, shall be added to the rerun list.'}]
```
Comment (seasidesparrow) [inline adsdocmatch/match_w_metadata.py:164]: If the input file fails parsing, the variable metadata will be undefined, and the return message itself will generate an exception.

Comment (golnazads) [reply adsdocmatch/match_w_metadata.py]: Fixed. Thank you.

**Comment by seasidesparrow** [comment]
In testing oracle updating, if there are no matches to be added to oracle, the update_db_curated_matches method in oracle_util will return an empty string.  Line #156 of run.py checks the value of status to determine whether to archive those spreadsheets.  It is possible that a curated spreadsheet may have no additional matches, in which case status will be interpreted as None, and spreasheet_util.archive(spreadsheet_filename) will not run, and the spreadsheet will remain in the "curated" folder where it will again be processed by oracle update and ignored.

oracle_util.py should return some string (e.g. "no matches found in curated file") in update_db_curated_matches at line 504.

**Comment by golnazads** [comment]
yes, that is what I shall do now.

**Comment by seasidesparrow** [comment]
Awesome, looks good!

**Pull request merged**